extensions/azurecore/package.json

```diff
@@ -28,6 +28,11 @@
             "type": "array",
             "default": null,
             "description": "%azure.resource.config.filter.description%"
+          },
+          "azure.tenant.config.filter": {
+            "type":"array",
+            "default": [],
```

Comment:
This is going to show up in the settings UI as well too, right? So users can easily remove tenants from the filter if needed?

Reply:
yes, but i think arrays are shown in the JSON only.

![image](https://user-images.githubusercontent.com/1465785/86838180-b5e9b680-c054-11ea-996a-d8c1b4cf8760.png)

Reply:
Maybe run the description by some other people too - I think something like 

`The list of tenant IDs to ignore when querying Azure resources` 

is a little bit clearer. But I'm not 100% happy with that.